src/transformStateWithClones.js

```diff
@@ -65,7 +65,25 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const arr = [];
+  const arrElem = { ...state };
```

Comment:
arr? I thought that an object.

Reply:
it's an object but it's also an arr's elem, that's why I called it in such a way
but know I think it's better to rename it